st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -67,18 +63,15 @@ def _match_tuple_to_dict(self, match):
             'representation': match[2]
         }
 
-    @request_user_has_permission(permission_type=PermissionType.ACTION_ALIAS_LIST)
-    @jsexpose()
     def get_all(self, **kwargs):
         return super(ActionAliasController, self)._get_all(**kwargs)
 
-    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_ALIAS_VIEW)
-    @jsexpose(arg_types=[str])
-    def get_one(self, ref_or_id):
-        return super(ActionAliasController, self)._get_one(ref_or_id)
+    def get_one(self, ref_or_id, requester_user):
```

Comment:
I personally liked the decorator approach more for checking permissions - it was more explicit and obvious.

I'm also fine with current approach as long as we have some "easy" way to (automatically) validate / ling that all the methods apply some permission check (previously that was easy to verify by checking that decorator was in place).